### PR TITLE
added definitions

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: tei-betamesaheft.xml [hunk 1]
```diff
@@ -489,32 +490,34 @@ End license statement: do not remove -->
                         <attDef ident="calendar" usage="opt" mode="replace">
                             <valList type="open">
                                 <valItem ident="world">
-                                    <desc>ʿĀmata ʿālam/ʿĀmata ʾəm-fəṭrat (Era of the World)</desc>
+                                    <desc>ʿĀmata ʿālam, ʿĀmata ʾəm-fəṭrat, ʿĀmat ʾəm-ʾAddām  (=Era of the World)</desc>
```
Comment (DenisNosnitsin1970) [inline tei-betamesaheft.xml:493]: ʿĀmat ʾəm-fəṭrat. "ʿĀmata məḥrat" can be added as the last in the string (sometimes it is used in the sense: creation of the world/Adam as the great mercy)

File: tei-betamesaheft.xml [hunk 2]
```diff
@@ -489,32 +490,34 @@ End license statement: do not remove -->
...
-                                    <desc> Era of Alexander</desc>
+                                    <desc> ʿĀmata ʾƎskəndər (= Era of Alexander of Macedon, year 1 = 319 BC)</desc>
                                 </valItem>
                                 <valItem ident="evangelists">
-                                    <desc>Evangelists' years</desc>
+                                    <desc>Years called after the Evangelists </desc>
                                 </valItem>
                                 <valItem ident="islamic">
-                                    <desc>Hiǧrī (Islamic)</desc>
+                                    <desc>ʿĀmata ʾagār (=Hiǧrī, Islamic Era)</desc>
```
Comment (DenisNosnitsin1970) [inline tei-betamesaheft.xml:512]: = Hiǧrī

File: tei-betamesaheft.xml [hunk 3]
```diff
@@ -489,32 +490,34 @@ End license statement: do not remove -->
...
-                                    <desc>ʿĀmata samāʿtāt (Era of Martyrs (Diocletian))</desc>
+                                    <desc>ʿĀmata samāʿtāt (Era of Martyrs/ Era of Diocletian, year 1 = 283/284 CE)</desc>
                                 </valItem>
                                 <valItem ident="alexander">
-                                    <desc> Era of Alexander</desc>
+                                    <desc> ʿĀmata ʾƎskəndər (= Era of Alexander of Macedon, year 1 = 319 BC)</desc>
                                 </valItem>
                                 <valItem ident="evangelists">
-                                    <desc>Evangelists' years</desc>
+                                    <desc>Years called after the Evangelists </desc>
```
Comment (DenisNosnitsin1970) [inline tei-betamesaheft.xml:509]: perhaps like this: Evangelist's name associated with the year (repeated in the canonical order: Matthew, Mark, Luke, John)

**Comment by thea-m** [comment]
@PietroLiuzzo , I'm sorry, again due to my transformation problems: could you please add these changes to the schema in the correct way?

**Pull request merged**